src/lib/validator.ts

```diff
@@ -260,3 +332,25 @@ export const validateACRName = (value: string): string | boolean => {
   }
   return true;
 };
+
+export const validateStorageKeVaultName = (value: string): string | boolean => {
```

Comment:
nit: `validateStorageKeVaultName` is missing a `y`, should be `validateStorageKeyVaultName`

Reply:
accept